docs/events.md

```diff
@@ -22,3 +22,18 @@ Currently the Fuzzy finder does not log any counter events.
 ## Standard events
 
 Currently the Fuzzy Finder does not log any standard events.
```

Comment:
I think we want to remove this `## Standard Events` section here, since it's included at the bottom of this file.

Reply:
ups, my mistake... copy pasta 😅